ui/components/app/transaction-list-item/transaction-list-item.component.js

```diff
@@ -334,20 +355,27 @@ function TransactionListItemInner({
           ///: END:ONLY_INCLUDE_IF
         }
         subtitle={
-          <TransactionStatusLabel
-            statusOnly
-            isPending={isPending}
-            isEarliestNonce={isEarliestNonce}
-            error={error}
-            date={date}
-            status={displayedStatusKey}
-            ///: BEGIN:ONLY_INCLUDE_IF(build-mmi)
-            custodyStatus={transactionGroup.primaryTransaction.custodyStatus}
-            custodyStatusDisplayText={
-              transactionGroup.primaryTransaction.custodyStatusDisplayText
-            }
-            ///: END:ONLY_INCLUDE_IF
-          />
+          isBridgeTx && isBridgeComplete === false ? (
+            <BridgeActivityItemTxSegments
+              bridgeTxHistoryItem={bridgeTxHistoryItem}
+              transactionGroup={transactionGroup}
+            />
+          ) : (
+            <TransactionStatusLabel
+              statusOnly
+              isPending={isPending}
+              isEarliestNonce={isEarliestNonce}
+              error={error}
+              date={date}
+              status={displayedStatusKey}
+              ///: BEGIN:ONLY_INCLUDE_IF(build-mmi)
+              custodyStatus={transactionGroup.primaryTransaction.custodyStatus}
+              custodyStatusDisplayText={
+                transactionGroup.primaryTransaction.custodyStatusDisplayText
+              }
+              ///: END:ONLY_INCLUDE_IF
+            />
+          )
         }
         rightContent={
```

Comment:
I think an override for `primaryCurrency` and `secondaryCurrency` needs to be passed for Bridging as well in order to display the correct amounts when bridging ERC20 tokens. Swaps and Swap+Send both do this

For example, the Bridge item in this ss should have shown the bridge amount `11.895 USDC` on the right column
![Screenshot 2024-11-19 at 3 19 05 PM](https://github.com/user-attachments/assets/81ac1207-bb2a-4b48-be4c-d3b506696a30)

Reply:
For this, it requires updates to the `TransactionController`, which I'm already in the process of making here: https://github.com/MetaMask/core/pull/4918/files

These changes will be consumed by this PR: https://github.com/MetaMask/metamask-extension/pull/28460

If we can just move this fix to 28460 that would be really helpful.